db/schema.rb

```diff
@@ -38,6 +38,14 @@
     t.index ["question_id"], name: "index_answers_on_question_id"
   end
 
+  create_table "cities", force: :cascade do |t|
+    t.string "name"
```

Comment:
can we have the same city name for the same state?

Reply:
done